src/equations/serre_green_naghdi_1d.jl

```diff
@@ -0,0 +1,960 @@
+@doc raw"""
```

Comment:
It's a bit hard to review this file when it is renamed. I would like to see a proper diff. Can you maybe rename this file back to "serre_green_naghdi_flat_1d.jl" for the moment until I reviewed it and in the end rename it back? Or alternatively, we could rename the file in a separate PR.

Reply:
Done in #136